Core/WebCacheManager.swift

```diff
@@ -113,10 +119,11 @@ extension WebCacheManager {
         dataStore = nil
 
         let uuids = await WKWebsiteDataStore.allDataStoreIdentifiers
+        let count = max(0, uuids.count - 1) // -1 because there should be a current store
```

Comment:
I would suggest more descriptive naming e.g. `count` -> "leftOverDataStoreCountBeforeClearing" (maybe also pixel param name)

And `ids.count` -> "leftOverDataStoreCountAfterClearing"